translation/en-US.ts

```diff
@@ -245,8 +248,10 @@ export default {
   apply_publish: 'Apply online',
   publish: 'Publish',
   offline: 'Offline',
-  sure_publish: 'Are you sure you want to publish {{name}}?',
-  sure_offline: 'do you want to go offline {{name}}',
+  sure_publish: ({ name }: Record<'name', string>) =>
+    textJoin('Are you sure you want to publish', name, '?'),
+  sure_offline: ({ name }: Record<'name', string>) =>
```

Comment:
```suggestion
  sure_publish: ({ name }: { name: string }) =>
    textJoin('Are you sure you want to publish', name, '?'),
  sure_offline: ({ name }: { name: string }) =>
```
单字段直接写**类型字面量**即可（下同）